src/__tests__/interpreter-errors.ts

```diff
@@ -13,6 +13,13 @@ test('Undefined variable error is thrown', () => {
   `).toMatchInlineSnapshot(`"Line 1: Name im_undefined not declared"`)
 })
 
+test('Undefined variable error is thrown - verbose', () => {
+  return expectParsedError(stripIndent`
+	"enable verbose";
+	im_undefined;
+	`).toMatchSnapshot()
```

Comment:
My reason for using snapshots rather than inline snapshots was for readability - I didn't want the JSON content to make the test suite files messy. That said, you raise a good point, so I'm considering changing the inline snapshots to regular snapshots instead. Thoughts?

Also I feel like your last suggestion would be adding unnecessary functionality, since the interpreter has no actual use for such a function other than in testing.

Reply:
Snapshot content should be treated as code, and in this case it's better to inline the snapshots for better viewing. Other snapshots are not inlined because of two reasons:

- they contain much much more information, so inlining would really hurt readability, and
- they act more as a regression test.

In this case we probably want to inline because we also want to test the behaviour.

I don't know if all tests currently follow this convention, but if you find any feel free to PR.

I agree with you @ the other suggestion. No need to overcomplicate the code.

Reply:
On another note, please tell me when you're ready for review again.